frontend/Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM node:10.18.0-buster-slim
+FROM node:14-buster-slim
```

Comment:
What are we upgrading here?

Reply:
That's the version of node. We're currently using 10.18 but this PR would update us to 14!

Reply:
Thanks sounds good :>